Documentation/go_client.md

```diff
@@ -0,0 +1,18 @@
+# etcd go client API
+
+This document describes using the etcd client Go package `github.com/coreos/etcd/client`.
+
+## Endpoint Selection Mode
+
+This section describes how to configure a policy of choosing a request destination node in etcd client programs. The choice of destination node for client requests affects latency and throughput performance of etcd clusters.
```

Comment:
`s/This section describes how to configure a policy of choosing a request destination node in etcd client programs. //`

(delete sentence; redundant now)